CONTRIBUTING.md

```diff
@@ -42,7 +42,95 @@ you have written a test like this, make sure it does not fail often by
 uncommenting the code that sets a random seed in `tests/main.cpp` and running
 your test many times.
 
-Information on how to build and run the tests is in the main README.md file.
+### Install Dependencies
+
+<details open>
+<summary>Linux</summary>
+
+Use your distributions' package manager to install the required dependencies. The commands shown below are for Ubuntu/Debian.
```

Comment:
If you want, you can also add Fedora/RHEL/CentOS package manager instructions:

```
yum install armadillo-devel cmake
```

And I think in both of these systems, ensmallen can be installed directly, via `libensmallen-dev` on Debian/Ubuntu and `ensmallen-devel` on Fedora/RHEL/CentOS.  I am not sure of the state of packaging on Arch.

Reply:
For Arch, I found [this](https://aur.archlinux.org/packages/ensmallen) package.

Reply:
Ah, right, Roberto packaged that.  If you want you can add the Arch command to install the dependencies or the library, but it's no huge deal---up to you.

Reply:
Sure, adding it too.